### PR TITLE
solution

### PR DESCRIPTION
**Pull request by HoTaBu4**
- [DEMO LINK](https://HoTaBu4.github.io/layout_stars/)
- [TEST REPORT LINK](https://HoTaBu4.github.io/layout_stars/report/html_report/)

- [x ] Yellow stars are added with container modifier + pseudo-selector (NO extra classes)
- [ x] Each BEM block has its own separate file
- [ x] All `Typical Mistakes` from `BEM` lesson theory are checked.
- [x ] Code follows all the [Code Style Rules ❗️](./checklist.md)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1 +1,40 @@
...
+    background-image: url("../src/images/star-active.svg");
+}
+
+.stars--4 .stars__star:nth-child(-1n + 4) {
+    background-image: url("../src/images/star-active.svg");
+}
+
+.stars--5 .stars__star:nth-child(n) {
+    background-image: url("../src/images/star-active.svg");
+}
```
Comment (Viktor-Kost) [inline src/style.css:40]:
```suggestion
.stars--1 .stars__star:nth-child(-1n + 1),
.stars--2 .stars__star:nth-child(-1n + 2),
.stars--3 .stars__star:nth-child(-1n + 3),
.stars--4 .stars__star:nth-child(-1n + 4),
.stars--5 .stars__star:nth-child(n) {
  background-image: url("../src/images/star-active.svg");
}
```

File: src/style.css [hunk 2]
```diff
@@ -1 +1,40 @@
-/* add styles here */
+* {
+    margin: 0;
+
+}
```
Comment (Viktor-Kost) [inline src/style.css:4]: it's not a good practice to use `*` selector for resetting margin. just reset for `body` tag
```suggestion
```

File: src/style.css [hunk 3]
```diff
@@ -1 +1,40 @@
-/* add styles here */
+* {
+    margin: 0;
+
+}
+
+.stars {
+    display: flex;
+}
```
Comment (Viktor-Kost) [inline src/style.css:8]:
```suggestion
.stars {
  display: flex;
}
```

File: src/style.css [hunk 4]
```diff
@@ -1 +1,40 @@
...
+    display: flex;
+}
+
+.stars__star {
+    width: 16px;
+    height: 16px;
+    background-repeat: no-repeat;
+    background-image: url("../src/images/star.svg");
+    background-position: center;
+}
```
Comment (Viktor-Kost) [inline src/style.css:16]:
```suggestion
.stars__star {
  width: 16px;
  height: 16px;
  background-repeat: no-repeat;
  background-image: url("../src/images/star.svg");
  background-position: center;
}
```

**Review by Viktor-Kost — CHANGES_REQUESTED** [review]
Also please check your deployment since current links lead to 404 page. If you need any help feel free to ask at fe_chat

**Comment by HoTaBu4** [comment]
https://hotabu4.github.io/layout_stars/report/html_report/ [TEST REPORT LINK]
https://hotabu4.github.io/layout_stars/ [DEMO LINK]

**Review by VitaliyBondarenko1982 — APPROVED** [review]
Link for tests not works but code and demo link with implementation looks good.

**Pull request closed without merging**